mackup/applications/calibre.cfg

```diff
@@ -0,0 +1,6 @@
+[application]
+name = Calibre
+
+[configuration_files]
+Library/Preferences/calibre
+.config/calibre
```

Comment:
Can you use the `xdg_configuration_files` for those? See https://github.com/lra/mackup/blob/master/doc/README.md#add-support-for-an-application-using-the-xdg-directory